### PR TITLE
Update KPI from PXP to pipeline runs

### PR DESCRIPTION
**Pull request by tayloramurphy**
Closes https://github.com/meltano/internal-data/issues/10

### CODE REVIEW

**Comment by tayloramurphy** [comment]
@pnadolny13 will you take an iteration on the implementation details?

**Comment by pnadolny13** [comment]
@tayloramurphy theres a definition of pipelines in https://handbook.meltano.com/data-team/metrics-and-definitions#pipeline that reflects the current state. Should I just put a cross reference there? Or were you looking for a different implementation using the active users definition we used in PxP i.e. >=7days old, 28 rolling window, etc.

**Comment by tayloramurphy** [comment]
@pnadolny13 I'm actually fine to cross-link if the logic is accurate. I can make that change and then get Douwe to approve and merge.

File: src/_data-team/kpis.md [hunk 1]
```diff
@@ -12,77 +12,19 @@ A Key Performance Indicator (KPI) is a way to measure and understand progress to
 
 Certain KPIs are considered Health Metrics, which means we actively monitor them and alarm on them and take action when and if they fall out of expected range. Otherwise we may or may not have goals or initiatives associated with their outputs.
 
-## Projects x Plugins Score (PXP)
+## Pipeline Runs
 
-Meltano's primary strategic goal is to achieve product-market fit for Meltano as the DataOps Platform Infrastructure.
-Our primary metric for measuring progress towards this goal is the number of projects times the amount of plugins categories used in each project.
+Meltano's primary strategic goal is to achieve product-market fit for Meltano as the DataOps Platform Infrastructure, with a particular focus on ELT workloads. Our primary metric for measuring progress towards this goal is the number of pipelines run.
```
Comment (DouweM) [inline src/_data-team/kpis.md:17]:
```suggestion
Meltano's primary strategic goal is to achieve product-market fit for Meltano as a data movement (EL+T) solution with 10X developer experience over alternatives. Our primary metric for measuring progress towards this goal is the number of pipelines run.
```

**Review by DouweM — CHANGES_REQUESTED** [review]
@tayloramurphy Changes look good, but it's time to restate the goal. Feel free to edit further.

**Comment by tayloramurphy** [comment]
@DouweM accepted your change and I think it's good as is. Please merge if you agree.

**Comment by DouweM** [comment]
@tayloramurphy Merged! Can you please share this in internal-announcements since it's a Data topic?

**Comment by tayloramurphy** [comment]
Inviting discussion from the team in https://github.com/meltano/internal-data/discussions/21

**Pull request merged**